modules/openapi-generator/src/main/resources/go-experimental/model.mustache

```diff
@@ -35,14 +52,14 @@ type {{classname}} struct {
 {{#description}}
 	// {{{description}}}
 {{/description}}
-	{{name}} *{{{dataType}}} `json:"{{baseName}},omitempty"{{#withXml}} xml:"{{baseName}}{{#isXmlAttribute}},attr{{/isXmlAttribute}}"{{/withXml}}{{#vendorExtensions.x-go-custom-tag}} {{{.}}}{{/vendorExtensions.x-go-custom-tag}}`
-{{#isNullable}}	isExplicitNull{{name}} bool `json:"-"{{#withXml}} xml:"-"{{/withXml}}`{{/isNullable}}
+	{{name}} {{^required}}*{{/required}}{{{dataType}}} `json:"{{baseName}}{{^required}},omitempty{{/required}}"{{#withXml}} xml:"{{baseName}}{{#isXmlAttribute}},attr{{/isXmlAttribute}}"{{/withXml}}{{#vendorExtensions.x-go-custom-tag}} {{{.}}}{{/vendorExtensions.x-go-custom-tag}}`
 {{/vars}}
 }
 {{/isEnum}}
 
 {{^isEnum}}
 {{#vars}}
+{{^required}}
 // Get{{name}} returns the {{name}} field if non-nil, zero value otherwise.
```

Comment:
This comment (and I think also the others on the generated methods) would be pretty confusing with the `Nullable*` structures. Could you change them to be more reflective of that?

Reply:
`// Get{{name}} returns the {{name}} field value if set, zero value otherwise`